[2024-03-16 00:00:19,263] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_rf_score 2024-03-15T23:59:00+00:00 [queued]>
[2024-03-16 00:00:19,293] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_rf_score 2024-03-15T23:59:00+00:00 [queued]>
[2024-03-16 00:00:19,294] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-16 00:00:19,294] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-16 00:00:19,294] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-16 00:00:19,311] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_rf_score> on 2024-03-15T23:59:00+00:00
[2024-03-16 00:00:19,323] {standard_task_runner.py:52} INFO - Started process 96109 to run task
[2024-03-16 00:00:19,328] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_rf_score', '2024-03-15T23:59:00+00:00', '--job-id', '5082', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpfxehp9ty', '--error-file', '/tmp/tmp853b2uzu']
[2024-03-16 00:00:19,329] {standard_task_runner.py:77} INFO - Job 5082: Subtask compute_rf_score
[2024-03-16 00:00:19,398] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_rf_score 2024-03-15T23:59:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-16 00:00:19,492] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_rf_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-15T23:59:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-15T23:59:00+00:00
[2024-03-16 00:00:22,057] {python.py:151} INFO - Done. Returned value was: None
[2024-03-16 00:00:22,072] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_rf_score, execution_date=20240315T235900, start_date=20240316T000019, end_date=20240316T000022
[2024-03-16 00:00:22,119] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-16 00:00:22,160] {local_task_job.py:151} INFO - Task exited with return code 0
